spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+RSpec.describe 'Post Show Page', type: :feature do
+  before do
+    # Set up any necessary data or visit relevant pages
+  end
+
+  it 'displays the post title' do
+    # Test code goes here
+  end
+
+  it 'displays the author of the post' do
+    # Test code goes here
+  end
+
+  it 'displays the number of comments the post has' do
+    # Test code goes here
+  end
+
+  it 'displays the number of likes the post has' do
+    # Test code goes here
+  end
+
+  it 'displays the post body' do
+    # Test code goes here
+  end
+
+  it 'displays the username of each commentor' do
+    # Test code goes here
```

Comment:
- [ ] Good to have this post-show test file. :+1:  But please from the project requirement, you should write a test for the following :point_down: But now you have no test code in your `it` block to test the following:blush: 

  * Tests that asserts you can see the post's title.
  * Tests that asserts you can see who wrote the post.
  * Tests that asserts you can see how many comments it has.
  * Tests that asserts you can see how many likes it has.
  * Tests that asserts you can see the post body.
  * Tests that asserts you can see the username of each commentor.
  * Tests that asserts you can see the comment each commentor left.